Source/Core/DolphinQt/Debugger/JitBlockTableModel.h

```diff
@@ -0,0 +1,130 @@
+// Copyright 2024 Dolphin Emulator Project
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#include <QAbstractTableModel>
+#include <QList>
+#include <QVariant>
+
+#include "Common/CommonTypes.h"
+#include "Common/Lazy.h"
+#include "Core/PowerPC/JitCommon/JitCache.h"
+
+namespace Core
+{
+enum class State;
+class System;
+}  // namespace Core
+class JitInterface;
+class PPCSymbolDB;
+class QString;
+
+namespace JitBlockTableModelColumn
+{
+enum EnumType : int
+{
+  PPCFeatureFlags = 0,
+  EffectiveAddress,
+  CodeBufferSize,
+  RepeatInstructions,
```

Comment:
I think the repeat instructions column will be 0 the vast majority of the time. Have you found it useful?

Reply:
JIT blocks with recompiled repeat instruction are an incredibly small minority indeed, but don't forget columns can be permanently hidden if not desired. I'm not sure for whom this information might be useful, but it was a simple feature to add and I thought it was neat.

Reply:
I suppose we could keep it then.